internal/service/dataexchange/event_action_test.go

```diff
@@ -0,0 +1,671 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package dataexchange_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
```

Comment:
These should be in the block below (standard library imports, a space, third party libraries).